packages/terra-text/src/Text.jsx

```diff
@@ -86,12 +81,8 @@ const Text = ({
     attributes.className,
   ]);
 
-  const textStyles = {
-    color,
-  };
-
   return (
-    <span {...attributes} style={{ ...textStyles, ...customProps.style }} className={TextClassNames}>
+    <span {...attributes} style={{ ...customProps.style }} className={TextClassNames}>
```

Comment:
Remove customProps.style spread, it's no longer needed.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2211/commits/35775cd63f5cf0ae778b8946046ff2e9a349eea6.